minion/minion-grpc/grpc-client/src/main/java/org/opennms/horizon/minion/grpc/SimpleReconnectStrategy.java

```
@@ -22,18 +25,24 @@ public SimpleReconnectStrategy(ManagedChannel channel, Runnable onConnect, Runna
     }
 
     @Override
-    public void activate() {
+    public synchronized void activate() {
+        if (reconnectTask != null) {
+            // Silently ignore additional activate requests if one is already in progress
```

Is there any value in knowing when we reached this block? a debug or a trace message?

(I'm not familiar at all with this to know, but I always worry a little bit when we silently ignore things, as those can be messy to debug when something odd happens)

---

It's normal/expected for this to be called multiple times, so nothing worth logging really.
Could add something on DEBUG/TRACE if we really want to know though.